src/vfs.c

```diff
@@ -340,3 +340,27 @@ esp_err_t frogfs_vfs_register(const frogfs_vfs_conf_t *conf)
     s_frogfs_vfs[index] = vfs;
     return ESP_OK;
 }
+
+esp_err_t frogfs_vfs_deregister(const char *base_path)
+{
+    assert(base_path != NULL);
+
+    for (int i = 0; i < CONFIG_FROGFS_MAX_PARTITIONS; i++)
+    {
```

Comment:
This formatting does not match the rest of the project.  Could you place non-function block open braces on the same line as the condition?